### DISCUSSION TITLE
Zepp Watchface Maker deleting uploaded slices

### DISCUSSION THREAD

**Discussion by dtreed85** [zepp-health/zeppos-forum#289] [Q&A]
How do you delete unwanted slices that have been uploaded to Zepp Watchface Maker app?  I see in the release notes something about a one click removal of all images, but can't find that either.

I think the app is confused and my guess is because of the number of slices.  I go to upload images and tell it a new slice, but when it completes, it labels it one of the other uploaded slices.  Then when I try to run it on my device it says that the images haven't been uploaded but i know that they have.

**Comment by sunnyBob** [comment]
You can remove all images or remove single image. if you remove single image, the name of new image must be same as deleted，because the order of the images is determined by the name of the picture. There is also a limit to the number of images， for example, 10 for fonts images, 7 for days of the week and 29 for weather.
![image](https://github.com/zepp-health/zeppos-forum/assets/18245566/f05e5254-291d-44e9-89ed-d0665fcb0adf)

**Reply by dtreed85** [reply]
Thanks.   I've seen and used those.  What I was really referring to I guess was slices of images.  How can they be removed?

Your comment about naming makes me wander if my issue is my naming scheme.  When I say new slice and then upload the number images, it gives it one of the existing slice names, even though it shouldn't.
